source/unified-test-format/schema-1.3.json

```diff
@@ -213,27 +262,100 @@
           "properties": {
             "command": { "type": "object" },
             "commandName": { "type": "string" },
-            "databaseName": { "type": "string" }
+            "databaseName": { "type": "string" },
+            "hasServerId": { "type": "boolean" }
           }
         },
         "commandSucceededEvent": {
           "type": "object",
           "additionalProperties": false,
           "properties": {
             "reply": { "type": "object" },
-            "commandName": { "type": "string" }
+            "commandName": { "type": "string" },
+            "hasServerId": { "type": "boolean" }
           }
         },
         "commandFailedEvent": {
           "type": "object",
           "additionalProperties": false,
           "properties": {
-            "commandName": { "type": "string" }
+            "commandName": { "type": "string" },
+            "hasServerId": { "type": "boolean" }
           }
         }
       }
     },
 
+    "expectedCmapEvent": {
```

Comment:
New `invalid` tests prefixed with `expectedCmapEvent-` and based on existing tests for `expectedCommandEvent-*`.